Containerfile.tests

```diff
@@ -0,0 +1,30 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+FROM tokman
+
+WORKDIR /src
+ADD https://raw.githubusercontent.com/packit/deployment/master/scripts/setupcfg2rpm.py /src/setupcfg2rpm.py
+COPY setup.cfg /src/
+
+RUN dnf install -y \
+    git \
+    findutils \
+    make \
+    python3-flexmock \
+    python3-gunicorn \
+    python3-pip \
+    python3-pytest \
+    sqlite \
+    $(python3 setupcfg2rpm.py setup.cfg) \
+    && dnf clean all
+
+RUN pip install --upgrade sentry-sdk[flask]
+
+COPY . /src/
+RUN pip install --no-deps .
+
+EXPOSE 8000
+VOLUME /secrets
+
+CMD ["./run.sh"]
```

Comment:
I'm pretty sure most of this is redundant.

Reply:
Yes!  Given that 'make build-test-image' is now an incremental change
based on 'make build', Containerfile.tests only really needs a little setup
+ additional installs for testing.